narwhal/worker/src/batch_maker.rs

```diff
@@ -267,6 +267,15 @@ impl BatchMaker {
             return None;
         }
 
+        #[cfg(feature = "benchmark")]
+        // NOTE: This log entry is used to compute performance.
+        tracing::info!(
```

Comment:
Can we log at debug level if not in benchmark mode? This will help debugging in production too. Or enable logging at debug level in benchmarks?

Reply:
Just added as a debug line that can be used in both prod and in the benchmarks